Dockerfile

```diff
@@ -1,54 +1,87 @@
-FROM ubuntu:18.04
-
-RUN apt-get update && \
-    apt-get install -y curl \
-                       git \
-                       python \
-                       python-pip \
-                       python-dev \
-                       autoconf \
-                       libtool \
-                       gawk
-RUN curl -O https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz
-RUN tar xvf go1.10.3.linux-amd64.tar.gz
+# Build stage
+FROM golang:1-alpine as builder
 
 WORKDIR /app
 
-COPY ./bin /app/bin
-COPY ./lib /app/lib
-COPY ./share /app/share
-COPY ./requirements.txt /app
-COPY ./src/we-lang/we-lang.go /app
+COPY ./share/we-lang/we-lang.go /app
+
+RUN apk add --no-cache git
 
-# There are several files that must be fetched/created manually
-# before building the image
-COPY ./.wegorc /root
-COPY ./.ip2location.key /root
-COPY ./airports.dat /app
-COPY ./GeoLite2-City.mmdb /app
 
-RUN export PATH=$PATH:/go/bin && \ 
+RUN true && \ 
     go get -u github.com/mattn/go-colorable && \
     go get -u github.com/klauspost/lctime && \
     go get -u github.com/mattn/go-runewidth && \
-    export GOBIN="/root/go/bin" && \
-    go install /app/we-lang.go
+    CGO_ENABLED=0 go build /app/we-lang.go
+# Results in /app/we-lang
+
+
+# FROM ubuntu:18.04
+# RUN apt-get update && \
+#     apt-get install -y curl \
+#                        git \
+#                        python \
+#                        python-pip \
+#                        python-dev \
+#                        autoconf \
+#                        libtool \
+#                        gawk
```

Comment:
This can be removed.

Reply:
Agree.

Thank you for reviewing it and feel free to submit a new pull request with the fixes.